### ISSUE TITLE
How to edit to make it in another language?

### ISSUE THREAD

**Issue by TheNomad11** [xitanggg/open-resume#109]
Great project, but English only. Is it possible to make CVs in other languages? Which files would i Have to edit? Thamks!

**Comment by Sahil-Sharma-23** [comment]
> Great project, but English only. Is it possible to make CVs in other languages? Which files would i Have to edit? Thamks!

Great feature idea @TheNomad11 
Internationalization  could be added to the project.

You can follow https://nextjs.org/docs/app/building-your-application/routing/internationalization this doc to add your preferred language to the project.

**Comment by knolljo** [comment]
> Great project, but English only. Is it possible to make CVs in other languages? Which files would i Have to edit? Thamks!

Did you manage to add another language? I'd be interested in that too! 

For more translations something like weblate or https://docs.codeberg.org/codeberg-translate could be used.

**Comment by TheNomad11** [comment]
> Did you manage to add another language? I'd be interested in that too!

No, it looks quite complex, and I'm no developer, unfortunately

**Comment by zain-bin-ramzan** [comment]
@Sahil-Sharma-23 
I have added a multi-language feature on [Payfll](https://pafyll.com/en-NO) and can also submit a pull request for this project. However, I'm not sure if the project maintainers are interested, as it could take one or two weeks to make the entire site multilingual. I'd like to get a green light from the team before starting work on it.

**Comment by zain-bin-ramzan** [comment]
@xitanggg

**Comment by Sahil-Sharma-23** [comment]
@abuzain432432 
@xitanggg doesn't seem to be much active on this repository. Even if you want to build the feature, go ahead!
You can always create a PR for the same and whenever @xitanggg is back, it can be reviewed and merged. But can't provide confirmation on that. It's your choice to do so.
